### DISCUSSION TITLE
Readme documentation request: How much RAM should be allocated, at a minimum, to Docker?

### DISCUSSION THREAD

**Discussion by pepper-lees** [nextcloud/all-in-one#2321] [Questions] [answered]
Especially for Windows, where WSL's memory can be hard-capped, it would be greatly beneficial to know how much memory Nextcloud needs to operate. If it's known, could it be included in the readme? I'm flying blind and I couldn't see any mention of minimum memory requirements in this documentation. Tried looking in discussions too, but couldn't find anything after a quick search. Thanks!

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one/discussions/1335

**Reply by pepper-lees** [reply]
Hi, maybe you could pin that discussion and/or put it into the main readme?

**Reply by szaimen** [reply]
It is pinned 🤔

**Reply by pepper-lees** [reply]
Oh, so you're right. I completely missed that. Wish Github didn't truncate the title ("General recommendations for...")

**Reply by FlattusBlastus** [reply]
If you add all the options, its 4GB. By default, WSL2 takes 50% of the total available RAM. Therefore, you can get by on a 8GB system if you run a fairly clean Windows host. Obviously, the optimal system is at least a 16GB box.
